### PR TITLE
Arcaflow Container Toolkit

### PR DESCRIPTION
**Pull request by jdowni000**
## Changes introduced with this PR

- This is the start of renaming everything that was `arcaflow-plugin-image-builder`/`carpenter` to `arcaflow-container-tookit`/`act`.
- These changes have been tested locally, however builds will fail in github until we merge these changes and make a new release as there are naming conflicts with published packages.
- Do keep in mind there was a recent update in the `arcaflow-reusable-workflow` repository where internal and external users may use current state as of today while we undergo this overhaul.
- Additionally, I have created a new `README.md` file to try and enhance new user experience using the tool

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,25 +1,25 @@
 FROM golang:1.18 AS builder
 COPY . /build
 WORKDIR /build
-RUN CGO_ENABLED=0 go build ./carpenter.go
+RUN CGO_ENABLED=0 go build ./act.go
 
 
 FROM quay.io/centos/centos:stream8
```
Comment (dustinblack) [inline Dockerfile:7]: Make sure you rebase to account for changes from #52

File: README.md [hunk 2]
```diff
@@ -1,125 +1,170 @@
...
-  - url: ghcr.io
-    username_envvar: "GITHUB_USERNAME"
-    password_envvar: "GITHUB_PASSWORD"
   - url: quay.io
-    username_envvar: "QUAY_USERNAME"
-    password_envvar: "QUAY_PASSWORD"
-    namespace_envvar: "QUAY_NAMESPACE"
+    username_envvar: "<QUAY_USERNAME>"
+    password_envvar: "<QUAY_PASSWORD>"
+    namespace_envvar: "<QUAY_NAMESPACE>"
```
Comment (dustinblack) [inline README.md:79]: We should probably note that these Quay values are secrets, not variables.

File: README.md [hunk 3]
```diff
@@ -1,125 +1,170 @@
...
+  push:
+    branches:
+      - "**"
+  release:
+    types:
+      - published
+
+jobs:
+  arcaflow-container-toolkit:
+    uses: arcalot/arcaflow-container-toolkit/.github/workflows/reusable_workflow.yaml@main
```
Comment (dustinblack) [inline README.md:138]: Is this supposed to be a repo path? Or should/can it point to a published action?

**Comment by dustinblack** [comment]
Please rebase on main to pick up #51 and #52

File: README.md (hunk 3 above)
Comment (jdowni000) [reply README.md:138]: This example is showing the reusable workflow which would yes be a path. The reusable workflow itself uses the published action. Utilizing the reusable workflow in this manner will still allow the user to take advantage of some automatic configurable's as noted in the documentation. If the published action is only required they can call it using the next section `Arcaflow Container Toolkit as an Action'

File: README.md (hunk 2 above)
Comment (jdowni000) [reply README.md:79]: This is the local config `act.yaml` file that is being used locally. This file would not be used by secrets in github. They would be free to fill this yaml file out and run ACT as an executable.

**Pull request closed without merging**